module/module.go

```diff
@@ -277,14 +359,62 @@ func readDeps(jsonDeps map[string][]string) []Dependency {
 	return deps
 }
 
+func readPackageInfo(
+	packageRoot string,
+	baseDirectory string,
+	targetGenDir string,
+	className string,
+	instanceDirectory string,
+	jsonConfig *JSONClassConfig,
+	dependencies []Dependency,
+) (*PackageInfo, error) {
+	qualifiedClassName := strings.Title(camelCase(className))
+	qualifiedInstanceName := strings.Title(camelCase(jsonConfig.Name))
+	defaultAlias := camelCase(strings.ToLower(qualifiedInstanceName)) +
+		qualifiedClassName
+
+	relativeGeneratedPath, err := filepath.Rel(baseDirectory, targetGenDir)
+	if err != nil {
+		return nil, errors.Wrapf(
+			err,
+			"Error computing generated import string for %s",
+			targetGenDir,
+		)
+	}
+
+	return &PackageInfo{
+		PackageName:           defaultAlias,
+		PackageAlias:          defaultAlias + "Static",
+		GeneratedPackageAlias: defaultAlias + "Generated",
+		PackagePath: filepath.Join(
```

Comment:
We dont want to use `filepath.Join()` for go package names as that will use `\\` on windows which is probably invalid ? 

We can use `path.Join()` for go package names